shared/util/forward-logs.desktop.js

```diff
@@ -97,6 +97,7 @@ function setupTarget () {
       }
     }
 
+    // $FlowIssue these can no longer be written to
```

Comment:
I hammered at this one for a while before `$FlowIssue`ing it. Because [these are defined using the module syntax](https://github.com/facebook/flow/blob/v0.36.0/lib/core.js#L788), flow no longer lets us set them. I tried creating a new `console` object and setting it on the `window`/`global`, but the browser engine didn't accept mutating the global in that way.

Seems like this approach will not type check going forward, which I'm mostly ok with because it's hacky. We could work around it by importing our own console object, but that'd add a lot of boilerplate.

Maybe we could override the `console` type definition? Feels hacky too.